release-content/0.14/release-notes/9135_Add_pipeline_statistics.md

```diff
@@ -1 +1,20 @@
-TODO
+When working on rendering features (in the engine and in projects), it's useful to know how expensive they are. 
+[Tracy](https://github.com/bevyengine/bevy/blob/main/docs/profiling.md) already lets us measure CPU time per system, but we didn't have a way of measuring how long the GPU spends on each render node.
+
+The [`RenderDiagnosticsPlugin`](https://dev-docs.bevyengine.org/bevy/render/diagnostic/struct.RenderDiagnosticsPlugin.html) provides this information,
+collecting both information about the CPU/GPU spent per node and [pipeline](https://sotrh.github.io/learn-wgpu/beginner/tutorial3-pipeline/) statistics such as the number of triangles drawn.
+This data is stored in Bevy's central [`DiagnosticsStore`](https://dev-docs.bevyengine.org/bevy/diagnostic/struct.DiagnosticsStore.html),
+and can be accessed by calling `DiagnosticsStore::iter()` to get a list of all registered diagnostics.
+The [`DiagnosticPath`](https://dev-docs.bevyengine.org/bevy/diagnostic/struct.DiagnosticPath.html) contained in the [`Diagnostic`](https://dev-docs.bevyengine.org/bevy/diagnostic/struct.Diagnostic.html)s returned can also be used to fetch specific diagnostics for more focused investigation.
+
+By default, this feature will track:
```

Comment:
I would also add that we can track individual render/compute passes, groups of passes (e.g. all shadow passes), and individual commands inside passes (like draw calls). For this to work, they have to be instrumented with calls to methods from `RecordDiagnostics` trait.

To summarize, render diagnostics aren't recorded automatically, nodes must explicitly record them. As far as I'm aware, not all nodes currently have this instrumentation.

Reply:
Yeah, this has no integration with nodes atm. You have to specify the rendering work you want to record.

I also don't think the details about the types are useful. It would be better to show an example of getting the diagnostic recorder, instrumenting a span of rendering commands, and then showing how the LogDiagnosticsPlugin can print the results to the terminal.